frontends/concrete-python/concrete/fhe/mlir/converter.py

```diff
@@ -57,46 +57,76 @@ def convert(
             MlirModule:
                 In-memory MLIR module corresponding to the graph
         """
-
-        self.process(graph)
```

Comment:
Why the change :sweat_smile:

Reply:
I think github had a glitch here, I did not remove that.